config.py

```diff
@@ -0,0 +1,87 @@
+"""
+Configuration settings for the Bucket List API.
+
+The definition of the different configuration settings is contained here:
+- Development Configuration
+- Testing Configuration
+- Production Configuration
+"""
+
+import os
+
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+
+class Config:
+    """
+    The definition of the global configuration is defined here.
+
+    Attributes such as SECRET_KEY are the same no matter the platform used.
+    """
+
+    SECRET_KEY = 'u35dfh___vv@$%%jkdsvjkhb___jdfnv93043klm__vdmkll55vd----__'
+    SQLALCHEMY_COMMIT_ON_TEARDOWN = True
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+    USE_TOKEN_AUTH = True
+    DEBUG = False
+    SSLIFY_SUBDOMAINS = True
+    DEFAULT_PER_PAGE = 20
+    MAX_PER_PAGE = 100
+
+    @staticmethod
+    def init_app(app):
+        """
+        Initialize the application.
+
+        This static method initializes the application using whatever
+        configuration the user has chosen.
+        """
+        pass
```

Comment:
This `init_app` method is not needed

```python
app.config.from_object(<env>)
``` 
☝️ should suffice to initialize your config just like you have done [here](https://github.com/andela-bolajide/BucketListAPI/blob/develop/app/__init__.py#L21)

This method and the call to it is redundant and should be removed